### DISCUSSION TITLE
Packages/com.unity.timeline/api/UnityEngine.Timeline.ControlTrack

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#80] [Notes]
# Packages/com.unity.timeline/api/UnityEngine.Timeline.ControlTrack

https://docs.unity3d.com/Packages/com.unity.timeline@latest/index.html?subfolder=%2Fapi%2FUnityEngine.Timeline.ControlTrack.html

**Comment by vertxxyz** [comment]
Implement [`ITimeControl`](https://docs.unity3d.com/Packages/com.unity.timeline@latest/index.html?subfolder=/api/UnityEngine.Timeline.ITimeControl.html) to allow components on GameObjects targeted by ControlTracks to receive a time callback.
